### PR TITLE
[FR] Add `Account restrictions`

### PR DESCRIPTION
**Pull request by Ganondorf-1**
French translation of osu! wiki - Account restrictions

### CODE REVIEW

File: wiki/Help_Centre/Account_Restrictions/fr.md [hunk 1]
```diff
@@ -0,0 +1,116 @@
...
+- **Votre historique est trop chargé.** Dans les cas de joueurs qui manquent aux règles de façon importante, ou de joueurs qui sont à nouveau restreints après avoir été autorisé à rejouer suite à un appel réussi, nous pourrions simplement conclure que votre historique est trop chargé pour rejoindre le jeu à l'heure actuelle. Si cela vous arrive, nous vous recommandons d'attendre **un très long moment** avant de renvoyer un appel, tout en vous assurant de ne pas enfreindre d'autres règles pendant cette période.
+
+## Mes mails sont ignorés / Je ne pense pas que mon appel a été traité de façon juste
+
+Si vous pensez que vos mails ne sont pas lus ou traités correctement par notre équipe de support, veuillez nous envoyer un mail à [appeal-review@ppy.sh](mailto:appeal-review@ppy.sh) contenant les informations suivantes:
```
Comment (DoryanR) [inline wiki/Help_Centre/Account_Restrictions/fr.md:92]:
```suggestion
Si vous pensez que vos mails ne sont pas lus ou traités correctement par notre équipe de support, veuillez nous envoyer un mail à [appeal-review@ppy.sh](mailto:appeal-review@ppy.sh) contenant les informations suivantes :
```

File: wiki/Help_Centre/Account_Restrictions/fr.md [hunk 2]
```diff
@@ -0,0 +1,116 @@
...
+
+- Pendant combien de temps vous avez été restreint
+- La principale raison pour laquelle vous pensez avoir été restreint
+- Tous les appels précédents que vous avez envoyés et qui ont été ignorés/refusés
+- Un bref résumé des raisons pour lesquelles vous n'êtes pas satisfait du résultat de votre appel (Si possible)
```
Comment (DoryanR) [inline wiki/Help_Centre/Account_Restrictions/fr.md:97]:
```suggestion
- Un bref résumé des raisons pour lesquelles vous n'êtes pas satisfait du résultat de votre appel (si possible)
```

File: wiki/Help_Centre/Account_Restrictions/fr.md [hunk 3]
```diff
@@ -0,0 +1,116 @@
...
+
+Tous les mails envoyés à l'adresse de revue d'appels seront examinés par un tiers qui n'est pas impliqué dans les opérations quotidiennes de l'équipe de support et seront signalés pour une révision ou davantage de revue.
```
Comment (DoryanR) [inline wiki/Help_Centre/Account_Restrictions/fr.md:99]:
```suggestion
Tous les mails envoyés à l'adresse de review d'appels seront examinés par un tiers qui n'est pas impliqué dans les opérations quotidiennes de l'équipe de support et seront signalés pour une révision ou davantage de reviews.
```

File: wiki/Help_Centre/Account_Restrictions/fr.md [hunk 4]
```diff
@@ -0,0 +1,116 @@
...
+
+Veuillez comprendre que l'envoi d'un mail à cette adresse ne garantit qu'une deuxième revue ou la réponse initiale à votre appel, cela ne garantit pas un changement dans les résultats si vous avez enfreint les attentes impliquées dans le processus d'appel.
```
Comment (DoryanR) [inline wiki/Help_Centre/Account_Restrictions/fr.md:101]:
```suggestion
Veuillez comprendre que l'envoi d'un email à cette adresse ne garantit qu'un second examen/une première réponse à votre appel, mais ne garantit pas un changement de résultat s'il s'avère que vous avez violé les attentes impliquées dans la procédure d'appel.
```

File: wiki/Help_Centre/Account_Restrictions/fr.md [hunk 5]
```diff
@@ -0,0 +1,116 @@
...
+
+**Nous ne répondrons qu'aux requêtes de revue d'appels envoyés à [appeal-review@ppy.sh](mailto:appeal-review@ppy.sh).** N'essayez pas de contacter les membres de l'équipe osu! (ou les membres des équipes bénévoles) via Twitter, Discord ou toute autre plate-forme afin de contourner cette politique - ils ne peuvent pas vous aider.
```
Comment (DoryanR) [inline wiki/Help_Centre/Account_Restrictions/fr.md:103]:
```suggestion
**Nous ne répondrons aux demandes d'examen d'appel ou d'enquête que par le biais d'emails envoyés à [appeal-review@ppy.sh](mailto:appeal-review@ppy.sh).** N'essayez pas de contacter les membres de l'équipe d'osu! (ou les membres des équipes de bénévoles) via Twitter, Discord ou toute autre plateforme afin de contourner cette politique - ils ne peuvent pas vous aider.
```

**Review by DoryanR — CHANGES_REQUESTED** [review]

File: wiki/Help_Centre/Account_Restrictions/fr.md (hunk 5 above)
Comment (geshii) [reply wiki/Help_Centre/Account_Restrictions/fr.md:103]: envoyées**

**Review by geshii — APPROVED** [review]

**Comment by DoryanR** [comment]
Before merge, I prefer to check one last time

File: wiki/Help_Centre/Account_Restrictions/fr.md [hunk 6]
```diff
@@ -0,0 +1,116 @@
...
+- **Vous avez triché.** L'utilisation de programme tiers, de macros, ou n'importe quelle autre forme d'avantage injuste mènera à une restriction immédiate.
+- **Autres cas.** L'équipe de support aux comptes se réserve le droit de restreindre un compte pour des cas qui ne font pas partie de cette liste.
+
+Si un compte a été restreint à juste titre, nous demanderons à l'utilisateur d'attendre un moment avant d'envoyer son appel. Cela s'appelle un *cooldown*, et sa durée varie selon les infractions qui ont été commises.
+
+## Que dois-je faire durant le cooldown ?
+
+- **Continuez à jouer !** Tant que votre compte est restreint et que vous avez un cooldown, vous êtes plus que bienvenu pour continuer à jouer (tant que vous jouez sur votre compte principal !). Les scores que vous réalisez pendant votre restriction seront bien pris en compte, et à moins que vos scores soient partiellement ou totalement supprimés, ils resteront lorsque la restriction sera levée.
```
Comment (DoryanR) [inline wiki/Help_Centre/Account_Restrictions/fr.md:44]:
```suggestion
- **Continuez à jouer !** Tant que votre compte est restreint et que vous avez un cooldown, vous êtes plus que bienvenue pour continuer à jouer (tant que vous jouez sur votre compte principal !). Les scores que vous réalisez pendant votre restriction seront bien pris en compte, et à moins que vos scores soient partiellement ou totalement supprimés, ils resteront lorsque la restriction sera levée.
```

File: wiki/Help_Centre/Account_Restrictions/fr.md [hunk 7]
```diff
@@ -0,0 +1,116 @@
...
+- **Par quel moyen vous l'avez fait.** Cela s'applique pour la triche - nous voulons savoir où vous avez obtenu l'outil que vous avez utilisé, comment vous l'avez trouvé, et s'il est toujours présent sur votre ordinateur ou non.
+- **Pourquoi nous devrions vous faire revenir.** La plupart du temps, ce point s'attache au second. Nous voulons nous assurer que vous ne recommencerez pas ! Comprenez que plus l'historique de votre compte contient de violations des règles, plus nous nous attendrons à ce que cette section de l'appel soit remplie.
+
+## Et si mon appel est accordé ?
+
+Si nous décidons de vous laisser revenir, nous vous demanderons que vous acceptiez certains termes avant de lever la restriction.
+
+- **Vous ne recommencerez pas.** Cela inclut la triche et le partage de comptes.
+- **Vous ne créerez pas d'autre compte, que ça soit pour vous ou quelqu'un d'autre.**
```
Comment (DoryanR) [inline wiki/Help_Centre/Account_Restrictions/fr.md:66]:
```suggestion
- **Vous ne créerez pas d'autre compte, que ça soit pour vous ou pour quelqu'un d'autre.**
```

File: wiki/Help_Centre/Account_Restrictions/fr.md [hunk 8]
```diff
@@ -0,0 +1,116 @@
...
+- **Vous avez continué à briser les règles tout en étant restreint.** Il n'y a aucune raison de lever la restriction d'une personne qui continue d'enfreindre les règles même après avoir été restreinte. Si vous continuez à commettre des infractions, le cooldown de votre compte sera réinitialisé à votre dernière infraction. Dans le cas d'infractions extrêmes ou continues, votre appel peut être rejeté de façon permanente.
+- **Votre historique est trop chargé.** Dans les cas de joueurs qui manquent aux règles de façon importante, ou de joueurs qui sont à nouveau restreints après avoir été autorisé à rejouer suite à un appel réussi, nous pourrions simplement conclure que votre historique est trop chargé pour rejoindre le jeu à l'heure actuelle. Si cela vous arrive, nous vous recommandons d'attendre **un très long moment** avant de renvoyer un appel, tout en vous assurant de ne pas enfreindre d'autres règles pendant cette période.
+
+## Mes mails sont ignorés / Je ne pense pas que mon appel a été traité de façon juste
```
Comment (DoryanR) [inline wiki/Help_Centre/Account_Restrictions/fr.md:90]:
```suggestion
## Mes mails sont ignorés/Je ne pense pas que mon appel a été traité de façon juste
```

**Review by DoryanR — CHANGES_REQUESTED** [review]

**Review by DoryanR — APPROVED** [review]

**Review by TicClick — APPROVED** [review]

**Pull request merged**